<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWidgetClass</class>
 <widget class="QMainWindow" name="AdminWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="sidebarWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>221</width>
      <height>421</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #5B6DF8;</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>174</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Digital Wallet</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUsers">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>221</width>
       <height>42</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #4C68FF;
    color: white;

    border-radius: 10px;
    text-align: left;
}
QPushButton:hover {
    background-color: #3B55D8;
}
</string>
     </property>
     <property name="text">
      <string>Users</string>
     </property>
     <property name="icon">
      <iconset resource="DigitalWalletSystem.qrc">
       <normaloff>:/DigitalWalletSystem/icons/User Male.png</normaloff>:/DigitalWalletSystem/icons/User Male.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="transactionsButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>221</width>
       <height>42</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #4C68FF;
    color: white;

    border-radius: 10px;
    text-align: left;
}
QPushButton:hover {
    background-color: #3B55D8;
}
</string>
     </property>
     <property name="text">
      <string>  Transactions</string>
     </property>
     <property name="icon">
      <iconset resource="DigitalWalletSystem.qrc">
       <normaloff>:/DigitalWalletSystem/icons/icons8-exchange-20.png</normaloff>:/DigitalWalletSystem/icons/icons8-exchange-20.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>221</width>
       <height>42</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #4C68FF;
    color: white;
    border-radius: 10px;
    text-align: left;
}
QPushButton:hover {
    background-color: #3B55D8;
}
</string>
     </property>
     <property name="text">
      <string>  Logout</string>
     </property>
     <property name="icon">
      <iconset resource="DigitalWalletSystem.qrc">
       <normaloff>:/DigitalWalletSystem/icons/icons8-logout-30.png</normaloff>:/DigitalWalletSystem/icons/icons8-logout-30.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-20</y>
      <width>881</width>
      <height>591</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Dashboard">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>831</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>701</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSave">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>460</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="My_Balalnce">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>831</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="pageTitle_3">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>My Balance</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="searchBox_2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 12px;
background-color: #f5f5f5;
color: black;
font: 10pt;</string>
         </property>
         <property name="placeholderText">
          <string>Search...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Transactions">
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>831</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="pageTitle_6">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>Transactions</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="searchBox_5">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 12px;
background-color: #f5f5f5;
color: black;
font: 10pt;</string>
         </property>
         <property name="placeholderText">
          <string>Search...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="My_Requests">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>831</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="pageTitle_4">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>My Requests</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="searchBox_3">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 12px;
background-color: #f5f5f5;
color: black;
font: 10pt;</string>
         </property>
         <property name="placeholderText">
          <string>Search...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Profile">
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>831</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="pageTitle_5">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>Profile</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="searchBox_4">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 12px;
background-color: #f5f5f5;
color: black;
font: 10pt;</string>
         </property>
         <property name="placeholderText">
          <string>Search...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DigitalWalletSystem.qrc"/>
 </resources>
 <connections/>
</ui>
